glustercli/cmd/peer.go

```diff
@@ -127,3 +131,38 @@ var poolListCmd = &cobra.Command{
 		peerStatusHandler(cmd)
 	},
 }
+
+// getPeerID return peerId of host
+func getPeerID(host string) (string, error) {
+	// Get Peers list to find Peer ID
```

Comment:
Add one more check to see the input is already peer id. (If Peer detach command is called by giving peer id itself)

    if uuid.Parse(host) != nil {
        return host, nil
    }

Reply:
do we need to compare that peerID with the available peer?
or let the server take care of it?

Reply:
added check